posthog/batch_exports/http.py

```diff
@@ -147,6 +149,17 @@ def retry(self, *args, **kwargs) -> response.Response:
             batch_export_run.data_interval_end,
         )
 
+        log_activity(
```

Comment:
definitely add a test since these are always tricky to get right the first time and then basically never change